src/elements/fields/PaymentMethodField.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { memo, useEffect } from 'react';
+import {
+  CardElement,
+  Elements,
+  useElements,
+  useStripe
+} from '@stripe/react-stripe-js';
+
+import { getStripePromise } from '../../integrations/stripe';
+import { filterStyleValues } from '../../utils/styles';
+import { filterKeys } from '../../utils/utils';
+
+import InlineTooltip from '../components/Tooltip';
+
+const stripeKey = 'any-key-does-not-matter-what';
```

Comment:
not sure if this was needed during dev, or still needed. would benefit from a comment explaining why you need a dummy key

Reply:
Yes it is still used in the dashboard only.  Adding a comment.